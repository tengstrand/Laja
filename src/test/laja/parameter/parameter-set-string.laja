
#write "target/test-output/parameter-set-string.out"
  message1 = {message1}
  message2 = {message2}
#end
