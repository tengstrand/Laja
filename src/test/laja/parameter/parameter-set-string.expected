  message1 = This is a string
  message2 = Hello
